Packages/com.chisel.editor/Chisel/Editor/Editor/MaterialBrowser/ChiselEmbeddedFonts.cs

```diff
@@ -0,0 +1,50 @@
+/* * * * * * * * * * * * * * * * * * * * * *
+Chisel.Editors.ChiselEmbeddedFonts.cs
```

Comment:
Not sure if this header makes sense? Why repeat the filename? Would make more sense to add the project title / url or something

Reply:
Changed this for all files that had it. now has:
project url
license
author
description